src/js/server/hcapi/applications.js

```diff
@@ -4,6 +4,32 @@ const { createHttpError } = require('./errors');
 
 const applicationsRouter = new Router();
 
+applicationsRouter.get('/', (req, res, next) => {
+  HackerApplication
+    .findAll({
+      include: [
+        {
+          model: Hacker,
+          required: true,
+        },
+      ],
+    })
+    .then(applications => Promise.all(applications.map(appl => appl.hacker.getResponseStatus(appl).then(responseStatus => ({
```

Comment:
I've just been testing with a few applications — and those seem to load fine, but I haven't tried with the full applicant list. The query that's being repeated a lot is quite small, so hopefully for the number of applicants we have, and the number of requests that are going to be made, it wouldn't be a problem. We could try it, and see — have you got any alternative suggestions as to how to avoid this?